src/arcsf/config/experiment_config.py

```diff
@@ -0,0 +1,164 @@
+import os
+import warnings
+from copy import copy
+from itertools import product
+
+import wandb
+import yaml
+from jinja2 import Environment, FileSystemLoader
+
+from arcsf.config.config_class import Config
+from arcsf.models.config_class import ModelConfig
+
+
+def _listify(obj: object):
+    if isinstance(obj, list):
+        return obj
+    return [obj]
+
+
+def generate_experiment_configs(
+    top_config_name: str,
+) -> None:
+
+    # Read in yaml file
+    with open(f"configs/experiment/{top_config_name}.yaml") as f:
```

Comment:
would be preferable for this not to make assumptions about what directory the script has been called from (i.e. this is assuming the experiment config is always created when running python from the parent `arc-selective-forgetting` directory)

Reply:
have switched to producing the absolute path